apps/component-date/fields/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const dateComponent = require('hof-component-date');
+
+module.exports = {
+  name: 'component-date',
+  'date-of-birth': dateComponent('date-of-birth', {
+    // before means a date cannot be in the future
+    validate: ['required', 'date', 'before']
```

Comment:
as discussed there is no date validator by default.  In fact, you get some weird shizzle when you just put in an invalid date and use before and required
<img width="838" alt="screen shot 2017-06-01 at 14 51 18" src="https://cloud.githubusercontent.com/assets/12494656/26683016/51cc713e-46da-11e7-8f57-e3b014f68fff.png">